include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -197,23 +222,36 @@ namespace NovelRT::Maths
                 *reinterpret_cast<const glm::vec2*>(&point);
         }
 
+        /**
+         * @brief Compare this vector to other vector by using epsilonValue vector as a tolerance for any deviation between the two vectors for comparison
+         * @returns Return true if comparison between this vector and other vector fall within the tolerance set by epsilonValue vector, otherwise false
+         */
         bool epsilonEquals(GeoVector2F other, GeoVector2F epsilonValue) const noexcept
         {
             return glm::all(glm::equal(*reinterpret_cast<const glm::vec2*>(this),
                                        *reinterpret_cast<const glm::vec2*>(&other),
                                        *reinterpret_cast<const glm::vec2*>(&epsilonValue)));
         }
 
+        /**
+         * @brief Returns a Vector with uniform component of zero
+         */
         static GeoVector2F zero() noexcept
         {
             return GeoVector2F::uniform(0);
         }
 
+        /**
+         * @brief Returns a Vector with uniform component of one
+         */
```

Comment:
I hope you're finding these useful, I appreciate I'm writing a lot of them!
```suggestion
        /**
         * @brief creates a new GeoVector2F instance with a uniform value of one.
         * @return a new GeoVector2F instance with all components set to one.
         */
```